easybuild/easyblocks/generic/bundle.py

```diff
@@ -321,8 +320,37 @@ def install_step(self):
                             new_val = path
                         env.setvar(envvar, new_val)
 
-            # close log for this component
-            comp.close_log()
+    def make_module_req_guess(self):
+        """
+        Set module requirements from all comppnents, e.g. $PATH, etc.
+        During the install step, we only set these requirements temporarily.
+        Later on when building the module, those paths are not considered.
+        Therefore, iterate through all the components again and gather
+        the requirements.
+
+        Do not remove duplicates or check for existance of folders,
+        as this is done in the generic EasyBlock while creating
+        the modulefile already.
+        """
+        # Start with the paths from the generic EasyBlock.
+        # If not added here, they might be missing entirely and fail sanity checks.
+        final_reqs = super(Bundle, self).make_module_req_guess()
+
+        for cfg, comp in self.comp_instances:
+            self.log.info("Gathering module paths for component %s v%s", cfg['name'], cfg['version'])
+            reqs = comp.make_module_req_guess()
+
+            try:
+                for key, value in sorted(reqs.items()):
+                    if isinstance(reqs, string_type):
+                        value = [value]
+                    final_reqs.setdefault(key, [])
+                    final_reqs[key] += value
```

Comment:
Let's be more explicit that we're dealing with a list here:
```suggestion
                    final_reqs[key].append(value)